server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -201,8 +201,16 @@ public enum MetadataVersion {
     // Add new fetch request version for KIP-951
     IBP_3_7_IV4(19, "3.7", "IV4", false),
 
+    // New version for the Kafka 3.8.0 release.
     // Add ELR related supports (KIP-966).
-    IBP_3_8_IV0(20, "3.8", "IV0", true);
+    IBP_3_8_IV0(20, "3.8", "IV0", true),
```

Comment:
My understanding from trunk is that we removed the reference to KIP-966. metadataChanged should be false.

Reply:
See https://github.com/apache/kafka/blob/206d0f809ad8438e9d89b7f731a8b995f952f0c0/server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java#L214

Reply:
I had this set to false in one of the commits, but then I reverted it back to true, because of rebasing.